src/Chirp.Razor/DBFacade.cs

```diff
@@ -0,0 +1,50 @@
+using System.Data;
+using Microsoft.Data.Sqlite;
+using SQLitePCL;
+
+namespace Chirp.Razor
+{
+    public class DBFacade
+    {
+        public static List<CheepViewModel> Read()
+        {
+            List<CheepViewModel> cheeps = new List<CheepViewModel>();
+            
+            // Initialize SQLite provider ChatGPT help
```

Comment:
Was it just guidance or is part of the code pasted in? Please indicate in comment.